resources/validations/src/ssp-additions.sch

```diff
@@ -0,0 +1,1278 @@
+                test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'fedramp-logo']]">A FedRAMP OSCAL
+                SSP must attach the FedRAMP Logo.</sch:assert>
+            <sch:assert
+                doc:attachment="§15 Attachment 2"
+                id="has-user-guide"
+                role="error"
+                test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'user-guide']]">A FedRAMP OSCAL
+                SSP must attach a User Guide.</sch:assert>
+            <sch:assert
+                doc:attachment="§15 Attachment 5"
+                id="has-rules-of-behavior"
+                role="error"
+                test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'rules-of-behavior']]">A FedRAMP
+                OSCAL SSP must attach Rules of Behavior.</sch:assert>
+            <sch:assert
+                doc:attachment="§15 Attachment 6"
+                id="has-information-system-contingency-plan"
+                role="error"
+                test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'information-system-contingency-plan']]">
+                A FedRAMP OSCAL SSP must attach a Contingency Plan</sch:assert>
+            <sch:assert
+                doc:attachment="§15 Attachment 7"
+                id="has-configuration-management-plan"
+                role="error"
+                test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'configuration-management-plan']]">
+                A FedRAMP OSCAL SSP must attach a Configuration Management Plan.</sch:assert>
+            <sch:assert
+                doc:attachment="§15 Attachment 8"
+                id="has-incident-response-plan"
+                role="error"
+                test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'incident-response-plan']]"> A
+                FedRAMP OSCAL SSP must attach an Incident Response Plan.</sch:assert>
+            <sch:assert
+                doc:attachment="§15 Attachment 11"
+                id="has-separation-of-duties-matrix"
+                role="error"
+                test="oscal:resource[oscal:prop[@ns = 'https://fedramp.gov/ns/oscal' and @name = 'type' and @value = 'separation-of-duties-matrix']]">
+                A FedRAMP OSCAL SSP must attach a Separation of Duties Matrix.</sch:assert>
+        </sch:rule>
+    </sch:pattern>
+
+    <sch:pattern>
+        <sch:title>Policy and Procedure attachments</sch:title>
+        <sch:title>A FedRAMP SSP must incorporate one policy document and one procedure document for each of the 17 NIST SP 800-54r4 control
+            families</sch:title>
+
+        <!-- TODO: handle attachments declared by component (see implemented-requirement ac-1 for an example) -->
```

Comment:
The consensus with the OSCAL developers appeared (twice) to be that `inventory-item`s should hold the related attributes and that "inheritance" should not be used. This did not have certitude, so we could restore inheritance were that to be the final resolution.

Reply:
I was confused because, in our last meeting, you acknowledged their advice and said something along the lines subsequently between you, Dan, and I that you would keep it anyway, since you saw it as unwise to throw it out. I was surprised, but you suggested it was not such a big deal since the code was implemented and there was no reason to roll it back.

Am I mistaken?

Reply:
I do not think you are mistaken. I will overlay component inheritance on the system inventory after a get a clean unit test on system-inventory-less-component-inheritance.

I do not think there is any other way to reduce the complexity+redundancy of inventory properties other than to use component inheritance. However, that will be a novel type of system inventory representation to FedRAMP reviewers. I presume, that a transform from OSCAL to traditional FedRAMP system inventory will be desired if not necessary.